1.10/networking/virtual-networks/index.md

```diff
@@ -5,73 +5,60 @@ feature_maturity: preview
 menu_order: 04
 ---
 
-DC/OS supports Container Networking Interface (CNI)-compatible virtual networking solutions, including [Calico](https://github.com/dcos/examples/tree/master/calico) and Contrail. DC/OS also provides a native virtual networking solution called DC/OS Overlay. 
+DC/OS supports Container Networking Interface (CNI)-compatible virtual networking solutions, including [Calico](https://github.com/dcos/examples/tree/master/calico) and Contrail. DC/OS also provides a native virtual networking solution called DC/OS Overlay.
```

Comment:
This paragraph is very specific to CNI, and we should move into the CNI section.